### ISSUE TITLE
Some cache is broken after "Added defers to cache.Set to avoid waiting for operation to complete"

### ISSUE THREAD

**Issue by antonsoroko** [elgatito/plugin.video.elementum#1003]
## Current Behavior

после https://github.com/elgatito/elementum/commit/a9d7e4ca24cf24c376a36d35c25888a3243c742b

на примере TraktProgressShows:
код идёт в WatchedShowsProgress, 
потом в WatchedShows
а там теперь кеш пишется 2 раза:
1. сначала в кеш пишет тело ответа на HTTP запрос - Request -> Do -> CacheWrite
2. а потом в WatchedShows пишем в тот же ключ структуры

соотв когда заходим в прогресс второй раз, то в Request.CacheRead читает в Body структуру и получаем в итоге nil.

надо или поудалять штуки типо
```
	if len(shows) != 0 {
		defer cache.
			NewDBStore().
			Set(cache.TraktShowsWatchedKey, &shows, cache.TraktShowsWatchedExpire)
	}
```
чтобы кешем было только тело запроса,

или убрать кеш тела запроса (как было раньше)

или зачем-то 

## Steps to Reproduce (for bugs)

1.
4.
5.
6.

## Your Environment

* Version used:

* Environment name and version:

* Operating System and version:

* Link to Kodi debug log:

**Comment by elgatito** [comment]
@antonsoroko Да, там один и тот же ключ для кеша использовался. Я фикс запушил.

**Comment by antonsoroko** [comment]
@elgatito ах, не заметил, что TraktShowsWatchedKey используется для PreviousWatchedShows.

но тогда может стоит удалить `cacheKey`
из https://github.com/elgatito/elementum/blob/63eb9cd185ddbe0062961ad155e69edaec03468d/trakt/trakt.go#L386
он уже не используется. чтобы вызов функции не вводил в заблуждение о том какой ключ используется. а то сейчас всё равно выглядит так, что ключ один, хотя на самом деле уже нет.
